### PR TITLE
Bookstore: styling 🎉 ✔️ 🚀 

### PR DESCRIPTION
**Pull request by Nizamuddin4493**
# Bookstore: styling 🎉 ✔️ 🚀 
Used the concept of state management.
Used store, actions and reducers in React.
Connected React and Redux.
Applied React best practices and language style guides in code.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+.singlBook {
+  display: flex;
+  justify-content: space-between;
+  margin: 15px 0;
+  padding: 20px;
+  background-color: #fff;
+  border-radius: 10px;
+}
+
```
Comment (HAMMAS-SALEEM) [inline src/index.css:61]: ### Your Design
your-design

### Template
design (copy)


### Circlular Progressbar
Screenshot from 2022-08-28 02-44-09

- [ ] The writer name should come after the book name as shown in the purple box of the [Template](https://user-images.githubusercontent.com/93039359/187049246-d189ea50-be0b-4658-8330-74e2355e82d4.png) above :arrow_up: Please make your writer name come after the book name to make your design look professional :+1:
- [ ] Hey you are missing the buttons shown in the green boxes of [Template](https://user-images.githubusercontent.com/93039359/187049246-d189ea50-be0b-4658-8330-74e2355e82d4.png) above :arrow_up: Please add these buttons to make your design look professional :+1:
- [ ] Hey you have made a very good design but i guess you forgot to add the circular progressbar. Please add it to make your design look professional :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @Nizamuddin4493 :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following GitFlow
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following Linters

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+.singlBook {
+  display: flex;
+  justify-content: space-between;
+  margin: 15px 0;
+  padding: 20px;
+  background-color: #fff;
+  border-radius: 10px;
+}
```
Comment (phelian23) [inline src/index.css:60]: - [ ] As requested by the previous reviewer, kindly ensure your design is as close to the template as possible.
Take a look at the way the book title and author are arranged, also, the `remove`, `edit`, and `comment` button below the names are missing. Kindly fix it.
Your design
Capture
Zemplin template
187049246-d189ea50-be0b-4658-8330-74e2355e82d4

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Nizamuddin4493,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Great job setting up the correct linters ✔️
Professional documentation ✔️
Meaningful commit messages ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Nizamuddin4493,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**